### ISSUE TITLE
Few suggestions to improve tzindex

### ISSUE THREAD

**Issue by gituser** [blockwatch-cc/tzindex#10]
1. Provide non-docker build mechanism, not everyone has a docker installed and want to use, some prefer using LXC or KVM or any other type of virtualization, so maybe just add default `make` target to build using go and another target to build docker, should be easy, e.g.:

```
BUILD_TARGET=tzindex
BUILD_VERSION=dev
BUILD_COMMIT=$(git rev-parse --short HEAD)
REPO=blockwatch.cc/tzindex
VERSION=$BUILD_VERSION-$BUILD_COMMIT

# clean everything
go clean

# update dependencies 
go mod download

CGO_ENABLED=0 go build -a -v -o ./${BUILD_TARGET} -ldflags "-w -X ${REPO}/cmd.VERSION=${BUILD_VERSION} -X ${REPO}/cmd.GITCOMMIT=${BUILD_COMMIT}" ${BUILD_TARGET}.go
strip ./${BUILD_TARGET}
```

2. Tag every stable release with a version and provide basic changelog. I see there is a `v005` on github as a latest release, maybe switch to different version notation, e.g. `v005-1.0` as I understand v005 means protocol number (protocol 005 stands for babylon) on tezos

3. Add shortcuts for `--v --vv --vvv` `-v -vv -vvv` as it's much easier to input and most linux/unix programs use one dash instead of two for verbose

4.  If you try to run `tzindex` without specifying `config.json` it will fail and try to access `https://127.0.0.1:8732` instead of the default one `http://127.0.0.1:8732` and it's very hard to figure out for beginners that there is an issue in http scheme as there is no option in help of `tzindex` to switch `tls` on or off. Would be nice to add `--tls` option in the help and also switch it off by default.

Thanks!

**Comment by echa** [comment]
Thanks,

- 1, 2 and 4 are resolved in v5.2.0
- 3 is GNU/Posix style and will not be fixed (it's part of an upstream dependency https://github.com/spf13/pflag)

**Comment by gituser** [comment]
hey @echa many thanks for the update!
